contracts/PartyToken.sol

```diff
@@ -5,18 +5,25 @@ pragma solidity 0.8.5;
 import {ERC20VotesComp} from "@openzeppelin/contracts/token/ERC20/extensions/ERC20VotesComp.sol";
 import {ERC20Permit} from "@openzeppelin/contracts/token/ERC20/extensions/draft-ERC20Permit.sol";
 import {ERC20} from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
-import {Pausable} from "@openzeppelin/contracts/security/Pausable.sol";
 
 /*
 Party Token
 by Anna Carroll
 */
-contract PartyToken is ERC20VotesComp, Pausable {
+contract PartyToken is ERC20VotesComp {
+    // ============ Events ==============
+    
+    event Unpaused(address account);
```

Comment:
don't need to emit the address of the account that un-pauses because we know it has to be PartyDAO multisig!

Reply:
I would also rename to `LockupEnded` or something like that :)